test/helpers/db.js

```diff
@@ -9,18 +9,10 @@ const Notification = require('../../src/models/db/notification').Notification
 const Fulfillment = require('../../src/models/db/conditionFulfillment').ConditionFulfillment
 
 exports.init = function * () {
+  yield knex.migrate.rollback(knexConfig).then()
```

Comment:
Oracle was throwing an error that broke migrations when I ran `migrate` after truncating the db. It was trying to recreate the indices, but they already existed. It seemed safest to just blow away the tables before re-running migrate, to ensure a fresh db.